### PR TITLE
OPENEUROPA-2976: [oe_paragraphs] Chart paragraph.

### PR DESCRIPTION
**Pull request by sergepavle**

### CODE REVIEW

File: modules/oe_paragraphs_chart/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Required contrib modules
+
+Before enabling this module, make sure that the following modules are present in your codebase by adding them to your
+`composer.json` and by running `composer update`:
+
+```json
+"require": {
+    "drupal/json_field": "~1.0.0-rc3"
+}
+```
```
Comment (22Alexandra) [inline modules/oe_paragraphs_chart/README.md]: oe_webtools has to be required as well.

File: modules/oe_paragraphs_chart/config/install/core.entity_form_display.paragraph.oe_chart.default.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+id: paragraph.oe_chart.default
+targetEntityType: paragraph
+bundle: oe_chart
+mode: default
+content:
+  field_oe_media:
+    weight: 0
+    settings:
+      match_operator: CONTAINS
+      match_limit: 10
```
Comment (ademarco) [inline modules/oe_paragraphs_chart/config/install/core.entity_form_display.paragraph.oe_chart.default.yml:16]: This value was added in Drupal 8.8, which means that if someone enables this module on 8.7 the schema validation will fail (or should at least). Even if this is an optional module, given the short time to live of 8.7 (8.9 is weeks away), let's require on the main project Drupal core 8.8.

**Comment by ademarco** [comment]
Also a rebase is needed.

File: modules/oe_paragraphs_chart/config/install/core.entity_form_display.paragraph.oe_chart.default.yml (hunk 2 above)
Comment (sergepavle) [reply modules/oe_paragraphs_chart/config/install/core.entity_form_display.paragraph.oe_chart.default.yml]: Done and also applied to other submodules.

**Pull request merged**